gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageConfiguration.java

```diff
@@ -103,6 +103,7 @@ public Builder blockSize(int value) {
       return this;
     }
 
+
     /** Creates a new instance, but does not destroy the builder. */
     public CloudStorageConfiguration build() {
```

Comment:
That's reasonable enough. I'll update all the javadocs in a PR after this one. I'll also be running google-java-format on the entire thing.